### PR TITLE
CONFIG/SPEC: Bump version to 1.10.0

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by yosefe** [comment]
@shamisp can you pls look?

File: ucx.spec.in [hunk 1]
```diff
@@ -301,6 +301,8 @@ process to map the memory of another process into its virtual address space.
 
 
 %changelog
+* Mon Feb 10 2020 Yossi Itigin <yosefe@mellanox.com> 1.10.0-1
```
Comment (shamisp) [inline ucx.spec.in:304]: You probably want to bump the date ?

Comment (yosefe) [reply ucx.spec.in]: fixed + added test

Comment (shamisp) [reply ucx.spec.in]: Does rpm build complains about duplicated entries ?

Comment (yosefe) [reply ucx.spec.in]: no

**Comment by shamisp** [comment]
👍

**Pull request merged**